Userland/Libraries/LibWeb/HTML/HTMLDialogElement.cpp

```diff
@@ -37,9 +55,14 @@ void HTMLDialogElement::show_modal()
 }
 
 // https://html.spec.whatwg.org/multipage/interactive-elements.html#dom-dialog-close
-void HTMLDialogElement::close(Optional<String>)
+void HTMLDialogElement::close(Optional<String> return_value)
 {
-    dbgln("(STUBBED) HTMLDialogElement::close()");
+    // 1. If returnValue is not given, then set it to null.
+    if (!return_value.has_value())
+        set_return_value(String {});
```

Comment:
I'm assuming it's OK to set `returnValue` to the empty string here, as the IDL attribute isn't nullable.